_posts/2020-09-30-intro-to-hive-connector.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title:  "A gentle introduction to the Hive connector"
+author: Brian Olsen
+excerpt_separator: <!--more-->
+---
+One of the most confusing aspects when starting Presto is the Hive connector.
+
+Typically, you seek out the use of Presto when you experience an intensely slow
+query turnaround from your existing Hadoop or Hive infrastructure. In fact, the
+genesis of Presto came about due to these slow Hive query conditions at Facebook
+back in 2012. So when you learn that Presto has a Hive connector, it can be
+rather confusing since you moved to Presto to circumvent the slowness of your
+current Hive cluster. Another common source of confusion is when you want to 
+query your data from your S3 compatible object storage such as AWS S3, MinIO, 
+and Google Cloud Storage. This too uses the Hive connector and if that confuses
+you, this post will explain why.
+
+To understand the origins and inner workings of Presto's Hive connector, you
```

Comment:
I think this should go below the fold (more)

Reply:
fair enough, I was thinking the image would provide a little bait for folks to click on it but the excerpt is already lengthy.